### PR TITLE
List blob for storage validation

### PR DESCRIPTION
**Pull request by patelchandni**

### CODE REVIEW

File: DiagnosticsExtension/Models/ConnectionStringValidator/StorageValidator.cs [hunk 1]
```diff
@@ -111,7 +111,11 @@ public Task<TestConnectionData> TestConnectionString(string connectionString, st
 
             CloudBlobClient client = storageAccount.CreateCloudBlobClient();
             client.GetServiceProperties();
-            client.ListContainers();
+            IEnumerable<CloudBlobContainer> containerList = client.ListContainers();
+            foreach (var container in containerList)
```
Comment (sidkri) [inline DiagnosticsExtension/Models/ConnectionStringValidator/StorageValidator.cs:115]: Could you also add a comment here explaining why listing blobs is necessary?

File: NuGet.Config [hunk 2]
```diff
@@ -3,5 +3,6 @@
   <packageSources>
     <clear />
     <add key="app-services-diagnostics-portal-nuget-feed" value="https://pkgs.dev.azure.com/app-service-diagnostics-portal/_packaging/app-services-diagnostics-portal-nuget-feed/nuget/v3/index.json" />
+    <add key="nuget" value="https://api.nuget.org/v3/index.json" />
```
Comment (puneetg1983) [inline NuGet.Config:6]: This might break the DevOps pipeline as I believe that doesn't allow multiple sources so revert this change

File: DaaS/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -37,5 +37,5 @@
 //[assembly: AssemblyVersion("1.0.0.0")]
 //[assembly: AssemblyFileVersion("1.0.0.0")]
 [assembly: AssemblyCopyright("")]
-[assembly: AssemblyVersion("3.1.22826.01")]
-[assembly: AssemblyFileVersion("3.1.22826.01")]
+[assembly: AssemblyVersion("3.1.22826.02")]
```
Comment (puneetg1983) [inline DaaS/Properties/AssemblyInfo.cs]: Update assembly versions in all projects and change it to `3.2.*`

File: NuGet.Config (hunk 2 above)
Comment (puneetg1983) [inline NuGet.Config]: Yup I double checked, this has broken the pipeline so definitely remove it

**Review by puneetg1983 — CHANGES_REQUESTED** [review]

File: NuGet.Config (hunk 2 above)
Comment (patelchandni) [reply NuGet.Config]: This is removed, thanks!

**Review by puneetg1983 — APPROVED** [review]

**Review by sidkri — APPROVED** [review]
